stix_shifter_modules/nozomi/README.md

```diff
@@ -0,0 +1,535 @@
+                    "is_public": false,
+                    "is_node_learned": true,
+                    "is_reputation_bad": false
+                },
+                "8": {
+                    "type": "mac-addr",
+                    "value": "02:02:02:02:02:02"
+                },
+                "9": {
+                    "type": "x-ibm-ttp-tagging",
+                    "extensions": {
+                        "mitre-attack-ext": {
+                            "technique_id": "T1200",
+                            "technique_name": "Hardware Additions",
+                            "tactic_name": "Initial Access"
+                        }
+                    },
+                    "name": "Hardware Additions",
+                    "kill_chain_phases": [
+                        {
+                            "kill_chain_name": "mitre-attack",
+                            "phase_name": "Initial Access"
+                        }
+                    ]
+                }
+            },
+            "first_observed": "2024-01-22T11:59:09.665Z",
+            "last_observed": "2024-01-22T11:59:09.665Z",
+            "number_observed": 1
+        }
+    ],
+    "spec_version": "2.0"
+}
+```
+### Types of Attributes
+
+| Type                      | Description                                | Example                                                                                                            |
+|---------------------------|--------------------------------------------|--------------------------------------------------------------------------------------------------------------------|
+| Dictionary fields         | A key value pair attributes.               | {"type_name": "Malformed traffic"}, {""severity": 10}                                                              |
+| List of dictionary fields | A list containing one or more dictionaries | "techniques": [{"id": "T1200", "name": "Hardware Additions"},{"id": "T1071","name": "Application Layer Protocol"}] |
+
+### Observations
+- Operators ==, !=  for some fields are translated to include?, !include?.(For example: x-ibm-ttp-tagging:name, process:parent_ref.binary_ref.name)
+   Example: [x-ibm-ttp-tagging:name != 'Execution'] translates to query=alerts | where properties !include? \"Execution\"
+- The JWT token expires 30 minutes after being created.
+- It is observed that Nozomi API fails if translated query string length is more than MAX_QUERY_LENGTH. To avoid this failure, query is split at 'OR' conditions and multiple requests are made to Nozomi to get the result. However, the part of query joined by 'AND' operator won't be split and 'Query length is too long or Invalid Query' error is returned if the length of that part is beyond MAX_QUERY_LENGTH.
+
+
+### Limitations
+- The page param is the page number to return, and count is the page dimension. If count is not provided, the default value is 10,000; if page is not provided, the default page number is 1.
```

Comment:
This will likely need to be updated if the paging is changed.

Reply:
Updated as per the suggestion.